components/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent, useState } from 'react'
+
+interface InputProps {
+  /**
+   * Input Placeholder
+   */
+  placeholder: string;
+  /**
+   * Type Placeholder
+   */
+  type: string;
+  /**
+   * Click handler
+   */
+  onChange?: () => void;
+}
+
+/**
+ * Input component for user interaction
+ */
+const Input: FunctionComponent<InputProps> = ({ placeholder, type, ...props }) => {
+  const [value, setValue] = useState('');
+
+  const handleChange = (event: { target: { value: React.SetStateAction<string>; }; }) => {
+    setValue(event.target.value);
+  };
+
+  return (
+    <input
+      type={type}
+      role="input"
+      data-cy="input"
+      placeholder={placeholder}
+      onChange={handleChange}
+      value={value}
+      className="border-solid border-2 border-primary-purple p-2 focus:border-primary-orange focus:outline-primary-orange focus:outline"
+    />
+  )
+}
+
+export default Input;
```

Comment:
We should always add a [new line](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=Each%20line%20should%20be%20terminated,it%20isn't%20newline%20terminated.) at the end of the file.